nixos/modules/services/hardware/dockd.nix

```diff
@@ -29,65 +31,131 @@ in
       default = "/etc/dockd/undocked.conf";
       description = ''
         Path to the file that will store the configuration of the
-         environment when the ThinkPad is removed from the dock.
+        environment when the ThinkPad is removed from the dock.
+
+        Make sure the file exists and is writable by the user of the session.
       '';
     };
 
     dockHookPath = mkOption {
       type = types.str;
-      default = "/etc/dockd/dock.hook";
+      default = "";
       description = ''
-        Path to the executable to execute when the ThinkPad is placed
-        on the dock.
-        Ignored if the "dockHook" option is set.
+        Path to the executable to run when the ThinkPad is placed on the dock.
+        If set, make sure the file exists and is executable by the user of the session.
+
+        Example: "/etc/dockd/dock.hook"
```

Comment:
For examples, you can use a `mkOption` argument attribute like `example = "/etc/dockd/dock.hook";`